### DISCUSSION TITLE
how do I switch to a previous commit

### DISCUSSION THREAD

**Discussion by rburgst** [gitbutlerapp/gitbutler#4513] [Q&A]
I have lots of commits in my virtual branch, however its unclear how I can go back to an earlier state without loosing my commits. 

so imagine the following history in my virtual branch

C->B->A

So I am at `C`, but i would like to temporarily switch to `B` (and go back to `C` later).

**Comment by Byron** [comment]
I don't think this is currently possible, but I think it's a great idea which could probably be made to work.
CC @krlvi to see what he thinks.
